src/gui/dialogs/logging.cpp

```diff
@@ -80,7 +81,7 @@ void tlogging::pre_show(twindow& window)
 		tgroup<std::string>& group = groups_[this_domain];
 
 		tgrid* this_grid = logger_box.get_row_grid(counter);
-		for(unsigned int iter = 0; iter < widget_id_.size(); iter++){
+		for(unsigned int iter = 0; iter < id_size; iter++){
```

Comment:
Again, as mentioned above, this variable name is misleading - it's a counter, not an iterator.